src/LinkDotNet.Blog.Web/ServiceExtensions.cs

```diff
@@ -26,6 +27,7 @@ public static IServiceCollection AddApplicationServices(this IServiceCollection
         services.AddScoped<IFileProcessor, FileProcessor>();
         services.AddScoped<ICurrentUserService, CurrentUserService>();
 
+        services.AddSingleton<AsyncKeyedLocker<string>>();
```

Comment:
This seems to be not needed anymore, or is it?

Reply:
Yes, can be removed. Leftover. Will do so later as am out.

Reply:
Done.